src/service/url-replacements-impl.js

```diff
@@ -47,27 +63,32 @@ export class UrlReplacements {
     /** @private {!RegExp|undefined} */
     this.replacementExpr_ = undefined;
 
-    /** @private @const {!Object<string, function(*):*>} */
+    /** @private @const {!Object<string, !ReplacementDef>} */
     this.replacements_ = this.win_.Object.create(null);
 
-    /** @private @const {function():!Promise<?AccessService>} */
+    /** @private @const {function(Window):Promise<(AccessService|null)>} */
     this.getAccessService_ = accessServiceForOrNull;
 
     /** @private @const {!Promise<?Object<string, string>>} */
     this.variants_ = variantForOrNull(win);
 
     /** @private {boolean} */
     this.initialized_ = false;
+
+    // NOTE: getService prevents multiple instances of same service across
+    // binaries (e.g. extensions) therefore ensures only one click listener.
+    this.registerClickListener_();
```

Comment:
@cramforce it will need to be an additional click listener if we want to capture phase.  SG?